commons/passive/google/google_places_info.avsc

```diff
@@ -0,0 +1,25 @@
+{
+  "namespace": "org.radarcns.passive.google",
+  "type": "record",
+  "name": "GooglePlacesInfo",
+  "doc": "Information of places where the user’s device is last known to be located along with an indication of the relative likelihood for each place.",
+  "fields": [
+    { "name": "time", "type": "double", "doc": "Device timestamp in UTC (s)." },
+    { "name": "timeReceived", "type": "double", "doc": "Device receiver timestamp in UTC (s)." },
+    { "name": "types", "type": ["null", {
+      "type": "array", "items": {
```

Comment:
Unfortunately, Avro arrays work very poorly with Kafka sink connectors. Could you choose the most dominant type instead, or make a fixed number of most suitable types, each with default null? (e.g. type1, type2, type3)?

Reply:
Okay, should I use an enum or a string as the type for each `Place.Type` element?